openff/toolkit/utils/_viz.py

```diff
@@ -0,0 +1,107 @@
+import uuid
```

Comment:
Good catch on moving `viz.py` to `_viz.py` - I like the idea to just make the whole file private rather than everything inside it.

Reply:
That idea is all @Yoshanuikabundi